db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170410221932) do
+ActiveRecord::Schema.define(version: 20170410215512) do
```

Comment:
I had to fast_forward on the same branch (no changes though!) and then migrate...